upstream-community-operators/ffdl/ffdl-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,166 @@
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
```

Comment:
Please narrow down the clusterPermissions for the operator to what the operator will need to run successfully

Reply:
Done